.github/workflows/server-ci.yml

```diff
@@ -50,12 +52,24 @@ jobs:
         override: true
         profile: minimal
     - uses: Swatinem/rust-cache@v1
+
+    - name: Start dependencies
+      run: docker-compose -f "server/docker-compose.yml" up -d --build
+
```

Comment:
Does the github actions linux have docker & docker-compose by default? I don't see an install step

Reply:
I think you need to maybe say "uses docker", though given that the tests pass, I'm inclined to say the answer is yes? :)

Reply:
`docker-compose` is installed in the actions image as shown here: https://github.com/actions/virtual-environments/blob/c8b63063441fb63048458f7907292a814c25189f/images/linux/ubuntu2004.json#L181